tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -4332,6 +4357,7 @@ node_modules/chrome-devtools-frontend/front_end/coverage/CoverageModel.js(230,25
 node_modules/chrome-devtools-frontend/front_end/coverage/CoverageModel.js(267,37): error TS2339: Property 'peekLast' does not exist on type '{ end: number; count: number; }[]'.
 node_modules/chrome-devtools-frontend/front_end/coverage/CoverageModel.js(268,21): error TS2345: Argument of type '{ end: number; }' is not assignable to parameter of type '{ end: number; count: number; }'.
   Property 'count' is missing in type '{ end: number; }' but required in type '{ end: number; count: number; }'.
+node_modules/chrome-devtools-frontend/front_end/coverage/CoverageModel.js(418,32): error TS2339: Property 'upperBound' does not exist on type 'any[]'.
```

Comment:
need to check on this one, but it's probably an intended result of skipping circular references

Reply:
yep. I guess chrome-devtools monkey-patches `upperBound` onto arrays, but of course we don't know about that.